src/fragmentarium/ui/display/Transliteration.tsx

```diff
@@ -149,6 +148,64 @@ function DisplayLine({
     ]
   )
 }
+function DisplayDollarAndAtLineWithParenthesis({
```

Comment:
@jlaasonen I could use an if-else block and one component to check the type instead of two components. But this would put extra logic within the component (there was a pull request about this)

Reply:
What pull request? Please provide a link.

Reply:
Leave it like this for now. We should try to get the card done.

Reply:
Leave it like this for now. We should try to get the card done.